### ISSUE TITLE
Testing Help Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4639]
# Page to Test:

Help Page - https://openlibrary.org/help/faq/about

# Steps

Steps for this task are outlined in #4521
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Sabreen-Parveen** [comment]
This page is good to go, no issues found.
